package.json

```diff
@@ -1251,7 +1251,7 @@
                   "%cmake-tools.configuration.cmake.debugConfig.console.externalTerminal.description%",
                   "%cmake-tools.configuration.cmake.debugConfig.console.newExternalWindow.description%"
                 ],
-                "description": "%cmake-tools.configuration.cmake.debugConfig.console.description%"
+                "markdownDescription": "%cmake-tools.configuration.cmake.debugConfig.console.description%"
```

Comment:
This one works then?

Reply:
The description shows up when I test the branch:

![image](https://user-images.githubusercontent.com/16143955/183142409-b7150d21-69bb-4198-8e21-12d56115e71c.png)

Reply:
👍